test/index.js

```diff
@@ -788,3 +793,85 @@ describe('function helpers', () => {
     )
   })
 })
+
+describe('custom headers', () => {
+  it('sets custom headers in the Netlify configuration', async () => {
```

Comment:
[sand] it doesn't look like the `async` keyword is needed here and in the test below, could this  be removed?